project/README.rst

```diff
@@ -210,7 +210,7 @@ Time Table Session 3
   * - 27
     - 14:12 - 14:24
     - `soulrrrrr <https://github.com/soulrrrrr>`__
-    - `Columnar Array <soulrrrrr/README.rst>`__:
+    - `Contigious Array <soulrrrrr/README.rst>`__:
```

Comment:
It would be good to fix the typo.  Could you please also update the row in the timetable sessions 3?

Reply:
Typo fixed. Isn't this row the row in the timetable sessions 3? Is there any other place need to be modified?

Reply:
YOu are right.  I misread.